src/components/phase/PhaseDescription.css

```diff
@@ -1,3 +1,3 @@
 .PhaseDescriptionText p {
-  font-size: x-large;
+  font-size: 2em;
```

Comment:
Why do we need this? Can we maybe get rid of the CSS styles?

Reply:
This will use the set font-size (in use-effect) as relative font-size, so it will change depending on the font size setting.